Makefile.old

```diff
@@ -69,6 +70,7 @@ SAIL_DEFAULT_INST += riscv_insts_zicbom.sail
 SAIL_DEFAULT_INST += riscv_insts_zicboz.sail
 SAIL_DEFAULT_INST += riscv_insts_zvbb.sail
 SAIL_DEFAULT_INST += riscv_insts_zvbc.sail
+SAIL_DEFAULT_INST += riscv_insts_zvknhab.sail
```

Comment:
This change seems unrelated to this PR.

Reply:
This `Makefile.old` is not really used for building (hence it's name); its only use is to manually run SMT property checks.   I wanted to make sure these still passed so I updated it.